packages/models-library/src/models_library/api_schemas_webserver/folders_v2.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime
+from typing import NamedTuple
+
+from models_library.basic_types import IDStr
+from models_library.folders import FolderID
+from models_library.users import GroupID
+from models_library.utils.common_validators import null_or_none_str_to_none_validator
+from models_library.workspaces import WorkspaceID
+from pydantic import Extra, PositiveInt, validator
+
+from ._base import InputSchema, OutputSchema
+
+
+class FolderGet(OutputSchema):
+    folder_id: FolderID
+    parent_folder_id: FolderID | None = None
+    name: str
+    created_at: datetime
+    modified_at: datetime
+    owner: GroupID
+
+
+class FolderGetPage(NamedTuple):
+    items: list[FolderGet]
+    total: PositiveInt
+
+
+class CreateFolderBodyParams(InputSchema):
+    name: IDStr
+    parent_folder_id: FolderID | None = None
+    workspace_id: WorkspaceID | None = None
```

Comment:
If I create with both None what happens? it goes to root workspace?
If I create inside my workspace? I guess there is an ID? If I create inside a folder which lives inside a workspace? 
Maybe add some documentation. This would help, thanks